RobotLocation[] locations   # Array of robots' AMCL poses